rules/linux/lnx_file_deletion.yml

```diff
@@ -0,0 +1,27 @@
+title: File Deletion
+id: 30aed7b6-d2c1-4eaf-9382-b6bc43e50c57
+status: stable
+description: Detects file deletion commands
+author: Ömer Günal, oscd.community
+date: 2020/10/07
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1070.004/T1070.004.md
+logsource:
+    product: linux
+    category: process_creation
+detection:
+    keywords:
+        - Commands|contains:
+          - 'rm '
+          - 'shred -u'
+          - 'rmdir'
+          - 'unlink'
+          - 'busybox rm -f *'
+          - 'find * -delete'
+    condition: keywords
```

Comment:
Hello @omergunal! Could you please update the `detection` section using the approach that we discussed in the other PRs of yours?